### PR TITLE
[[ Bug 17690 ]] Add 8.0 Legacy option to Save as... file dialog

### PR DESCRIPTION
**Pull request by livecodeian**
Note: depends on LiveCode PR: https://github.com/livecode/livecode/pull/4280

### CODE REVIEW

File: Toolset/libraries/revidelibrary.8.livecodescript [hunk 1]
```diff
@@ -7433,30 +7433,47 @@ on revIDEActionSaveStackAs pStackID, pSubstackOnly
...
+   
    local tVersion
    switch 
       case the scriptOnly of stack tShortName
          put "script" after tStackName
          ask file revIDELocalise("Save script only stack %1 as:", tSubstitutions) with tStackName with type (revIDELocalise("LiveCode Script Only Stack") & "|livecodescript|RSTK")
          break
+	  case tStackFileType is 8.0
+	     ask file revIDELocalise("Save stack %1 as:", tSubstitutions) with tStackName with type tStack80Filter or type tStackFilter or type tStack70Filter or type tStack55Filter or type tStack27Filter or type tStack24Filter
+		 break
```
Comment (montegoulding) [inline Toolset/libraries/revidelibrary.8.livecodescript:7452]: @livecodeian I'm wondering what's caused the odd indenting here

**Comment by livecodeali** [comment]
Aside from the odd indenting,
@livecode-vulcan review ok 292dcc3

**Comment by livecode-vulcan** [comment]
💙 review by @livecodeali ok 292dcc3c8ef72478ed3da7ed9980c8a0c6549ab7

**Comment by macphailian** [comment]
Odd indenting probably due to TextWrangler replacing spaces with tabs. Sorry about that!

**Pull request merged**